src/sphinx_antsibull_ext/roles.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2021
+'''
+Add roles for semantic markup.
+'''
+
+from __future__ import (absolute_import, division, print_function)
```

Comment:
Does this code have to work on python 2?

Reply:
No, but the other files in this directory also have it.

Reply:
And gone it is :)